### PR TITLE
2019 Contributor Summits placeholders added

### PR DESCRIPTION
**Pull request by idvoretskyi**
Signed-off-by: Ihor Dvoretskyi <ihor@linux.com>

### CODE REVIEW

**Comment by mrbobbytables** [comment]
ref #3190 
/area contributor-summit

File: events/2019/11-contributor-summit/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# 2018 Kubernetes Contributor Summit NA
```
Comment (nikhita) [inline events/2019/11-contributor-summit/README.md]: This should have been 2019? :stuck_out_tongue:

Comment (idvoretskyi) [reply events/2019/11-contributor-summit/README.md]: LOL, yes, fixed here https://github.com/kubernetes/community/pull/3248 😄

**Pull request merged**